dotCMS/src/main/java/com/dotcms/storage/StorageKey.java

```diff
@@ -0,0 +1,113 @@
+package com.dotcms.storage;
+
+/**
+ * Encapsulates a storage key, it has the key (for instance the path)
+ * the group (for instance the bucket, folder, space, etc)
+ * Storage {@link StorageType}
+ * @author jsanca
+ */
+public class StorageKey {
```

Comment:
should be serializable?

Reply:
not really. it just a param passed to an API never used to store anything in cache. But I'll make it serializable it wouldn't hurt either